views/administration/helpdesk.hbs

```diff
@@ -24,11 +24,11 @@
                     Sende Wünsche oder Probleme direkt an den {{theme.short_title}}-Nutzer-Support.
                     <p>
                         <a href="/help?activeForm=team#contact-form" class="btn btn-primary">
-                        Support kontaktieren
+                        {{$t "administration.button.contactSupport" }}
```

Comment:
```suggestion
                        {{$t "administration.helpdesk.button.contactSupport" }}
```
because this text is visible on `/administration/helpdesk`